streams/src/test/java/org/apache/kafka/streams/processor/internals/ProcessorContextImplTest.java

```diff
@@ -200,7 +200,7 @@ public void localKeyValueStoreShouldNotAllowInitOrClose() {
             assertEquals(VAL, store.approximateNumEntries());
         });
     }
-
+    @SuppressWarnings("deprecation")
```

Comment:
It might be nice to add a comment justifying the suppression (namely, that we need to continue verifying `put(k,v)`, as long as it's still in the code base, and that we can remove the suppression when we remove the method)

Reply:
```suggestion
    @SuppressWarnings("deprecation")// continuing to support WindowStore#put(K, V) as internal api
```